### ISSUE TITLE
[Bug] scaled TV can't be placed properly

### ISSUE THREAD

**Issue by JohnMetroid** [DeathWrench/ScaleableTelevision#5]
sometime after 1.0.7, lost ability to place the scaled TV properly (e.g. on top of the file cabinet).  Updated to 2.0.1 again today to see if it was fixed, but the placement bug is still present.

1.0.7 is fine

**Comment by DeathWrench** [comment]
What values are you using? Cause I can scale it and place it just fine.
![image](https://github.com/DeathWrench/ScaleableTelevision/assets/45341450/02b25e41-1558-4bb7-80a5-013a2ce7e4bd)

**Comment by JohnMetroid** [comment]
Sorry for the incredibly late response, I must have missed the notification for this.

I checked out 2.0.1 again this morning and made a new config, and it seems like it's working fine.  Closing this out.  Will update later if anything seems to be misbehaving

![image](https://github.com/user-attachments/assets/0c4fe04f-a695-4ac1-9d86-65520e85812a)

**Comment by JohnMetroid** [comment]
Yeah, it looks like the issue I was having before is still there.  Certain sizes have placement problems.  The value above should have been something like:

![image](https://github.com/user-attachments/assets/9b76ac02-cfac-4c54-bbe7-9534d41f7475)

I then updated to  

![image](https://github.com/user-attachments/assets/8d38656f-b811-4945-9528-e14a0e363012)

and the TV can't get placed anywhere, even on the ground

![image](https://github.com/user-attachments/assets/c451f83a-8db0-4109-b1eb-21fdca4f09ff)
![image](https://github.com/user-attachments/assets/6f4a54a6-bef2-4023-8cd1-045217845c8a)

edit: any Y size above 1.82 causes the TV to be unplaceable.  I suspect it may be related to an earlier issue where the collision mesh of the object is not 1:1 with the visuals, and beyond this limit it's trying to place an object below the cabinet/floor collision zone

https://github.com/DeathWrench/ScaleableTelevision/issues/1

**Comment by DeathWrench** [comment]
You should use Lethal Config, you can edit the size of the TV in game with it.

**Comment by DeathWrench** [comment]
I think it's the Z coordinates that bug it out.

**Comment by JohnMetroid** [comment]
> You should use Lethal Config, you can edit the size of the TV in game with it.

I've been using Lethal Config, and I confirmed any Y value above 1.82 will prevent placement of the TV.  Completely reproducible

https://i.imgur.com/sbO6Ra9.gif

**Comment by JohnMetroid** [comment]
the TV collider has other bugs...  you can crank it up to a huge size (e.g. x = 5.4) and the collision on the right is much further than the actual television.

Apparent hitbox illustrated below

![image](https://github.com/user-attachments/assets/a7774bb2-37dc-404f-adc4-a2abff7cd8f8)
